dev/travis/before_script.sh

```diff
@@ -76,4 +76,20 @@ case $TEST_SUITE in
 
         cd ../../..
         ;;
+    js)
+        curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.1/install.sh | bash
+        export NVM_DIR="$HOME/.nvm"
+        [ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" # This loads nvm
+        nvm install --lts
```

Comment:
If I understood the doc right v7 is not LTS and will not be used in such code.

The point is that I don't see any value in dynamic `Node.js` version, it can only bring us some headache when the LTS version will switch. Explicit is better than implicit.

As soon as `v8` stable is released we can change value in our config if the tests still pass. Current version is like specifying `php: stable` version instead of concrete value.